src/modules/Dropdown/Dropdown.js

```diff
@@ -755,6 +756,38 @@ export default class Dropdown extends Component {
     return _.findIndex(options, ['value', value])
   }
 
+  getDropdownAriaOptions = (ElementType) => {
+    const { loading, disabled, search, multiple } = this.props
+    const { open } = this.state
+    const ariaOptions = {
+      role: 'listbox',
+      'aria-busy': loading,
+      'aria-disabled': disabled,
+      'aria-expanded': !!open,
+    }
+    if (search) {
+      ariaOptions.role = 'combobox'
+    } else if (ElementType === Menu.Item) {
+      ariaOptions.role = 'menuitem'
+    } else {
+      ariaOptions['aria-multiselectable'] = multiple
+    }
+    return ariaOptions
+  }
+
+  getDropdownMenuAriaOptions() {
+    const { search, multiple } = this.props
+    const ariaOptions = {}
+
+    if (search) {
```

Comment:
Same note here, this might need to be `select` so that we're differentiating between a "menu only" style dropdown and a `<select />` element style Dropdown.

http://react.semantic-ui.com/modules/dropdown#selection
^ Not searchable, but is a "select".

http://react.semantic-ui.com/modules/dropdown#search-selection
^ Exactly the same as a "select" Dropdown above, but you can search for items.

http://react.semantic-ui.com/modules/dropdown#dropdown
^ Regular dropdown like a menu at the top of an desktop application.